src/components/Project/index.js

```diff
@@ -137,17 +143,16 @@ class Project extends React.Component {
     }
 
     // count deployable features by comparing currentRelease created and feature created
-    var deployableFeatures = 0
-    if(project.currentRelease !== null){
-      project.features.entries.map(function(feature){
-        if(new Date(feature.created).getTime() >= new Date(project.currentRelease.headFeature.created).getTime()){
-          deployableFeatures += 1
-        }
-        return null
-      })
-    } else {
-      deployableFeatures = project.features.entries.length
-    }
+    let deployableFeatures = project.features.entries.length
+    if(project.currentRelease){
+      deployableFeatures = project.features.entries.filter(function(feature){
+        return (new Date(feature.created).getTime() > new Date(project.currentRelease.headFeature.created).getTime())
+      }).length
+    } 
+
+    let releasesInProgress = project.releases.entries.filter(function(release){
```

Comment:
we should change this var name to releasesRunning and filter for the state "running". Waiting releases are not relevant to know which ones are "in progress" since they haven't begun execution.